docs/getting-started.md

```diff
@@ -1,56 +1,56 @@
 This tutorial will guide you through the steps needed to install and
-run ShellHub on your local computer.
+run ShellHub.
 
 If you are looking for instructions on how to set up ShellHub
 for your device fleet on your cloud infrastructure the same steps apply.
 
 ## Installing
 
-ShellHub is built using the microservices design pattern, meaning that
+ShellHub is built using the microservices design pattern in the sense that
```

Comment:
I'd probably use something as: `ShellHub is designed using the microservices design pattern so multiple small and isolate services collaborate to provide the server.`